lsp-release.sh

```diff
@@ -51,8 +93,18 @@ function tmp_clone() {
 LSP_DIR=$(tmp_clone flux-lsp)
 UI_DIR=$(tmp_clone ui)
 
+function tag_lsp_release() {
+	v=v`version $LSP_DIR/Cargo.toml`
+	git tag -a -s $v -m "Release $v"
+	git push origin master $v
+	flux_version=$(grep -m 1 'flux = ' $LSP_DIR/Cargo.toml | version)
+	hub release create $v -m "Release $v
+
+	- Upgrade to [Flux $flux_version](https://github.com/influxdata/flux/releases/tag/$flux_version)" -e
+}
+
 cd $LSP_DIR
-lsp_version=$(grep -Eom 1 "([0-9]{1,}\.)+[0-9]{1,}" Cargo.toml)
+lsp_version=$(version Cargo.toml)
```

Comment:
Where does `version` come from? I don't have that installed here.

Reply:
It's a function defined at the top of this script.